recipe/build.sh

```diff
@@ -1,5 +1,21 @@
 #!/bin/bash
 
+# Avoid Xcode
+if [[ $(uname) == Darwin ]]; then
+  PATH=${PREFIX}/bin/xc-avoidance:${PATH}
+fi
+
+# Fix bison/m4 error
+# https://github.com/conda-forge/bison-feedstock/issues/7
+export M4=${BUILD_PREFIX}/bin/m4
+
+if [[ ${HOST} =~ .*linux.* ]]; then
+  # Missing g++ workaround.
+  ln -s ${GXX} g++ || true
+  chmod +x g++
+  export PATH=${PWD}:${PATH}
+fi
+
 qmake
-make -j $CPU_COUNT
```

Comment:
And maybe just calling `cmake` directly on Linux https://github.com/qtwebkit/qtwebkit/wiki/Building-QtWebKit-on-Linux ?